src/checkpoint/graphql/resolvers.ts

```diff
@@ -1,6 +1,14 @@
 import mysql from '../mysql';
+import { Logger } from '../utils/logger';
 
-export async function queryMulti(parent, args, context, info) {
+/**
+ *
+ */
```

Comment:
I'll add more description here, when in a follow up PR when I also pass the mysql instance through the context.